homework11/homework11/Order.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace homework11
+{
+    public class Order
+    {
+        public string ProductName;
+        public readonly int VendorCode;
+        public string СourierSurname;
+        public readonly int RequestNumber;
+        public DateTime DeliveryTime;
+        public DType OrderType;
+        public Order(string productname, int vendorcode, string couriersurname, int requestnumber, string deliverytime, DType ordertype)
+        {
+            ProductName = productname;
+            VendorCode = vendorcode;
+            СourierSurname = couriersurname;
+            OrderType = ordertype;
+        }
+        public override string ToString()
+        {
+            return $" Имя товара:{ProductName}. Артикул: {VendorCode}.";
```

Comment:
Зачем пробел в начале строки?
```suggestion
            return $"Имя товара:{ProductName}. Артикул: {VendorCode}.";
```